src/scripts/tray.js

```diff
@@ -87,23 +98,63 @@ function createAppTray () {
     };
 }
 
+function setTitle (title) {
+    switch (title) {
+        case "":
+            mainWindow.tray.setImage(_iconTray);
+            break;
+        case "•":
+            mainWindow.tray.setImage(_iconTrayTitleDot);
+            break;
+        case "1":
```

Comment:
Instead of having all these cases could you just generate the icon name by just using the title for all numbers e.g `` `baseName${title}.png` ``